internal/usage/usage.go

```diff
@@ -70,7 +70,7 @@ const (
 	MetricMode                      = "Atlas computes the current metric value as an average."
 	NotificationToken               = "Slack API token or Bot token or Flowdock personal API token." //nolint:gosec // This is just a message not a password
 	NotificationsChannelName        = "Slack channel name. Required for the SLACK notifications type."
-	APIKey                          = "Datadog API Key, Opsgenie API Key, VictorOps API key."
+	APIKey                          = "API Key."
```

Comment:
can we not loose the old description? we can rename the old to "AlertConfigAPIKey` if you want but I think is important to keep the old description as well as the new one